### PR TITLE
fix threejs `ShapeUtils`

### PR DESCRIPTION
**Pull request by FishOrBear**
ref: https://github.com/mrdoob/three.js/blob/dev/src/extras/ShapeUtils.js

### CODE REVIEW

**Review by dhritzkiv — APPROVED** [review]

File: types/three/three-core.d.ts [hunk 1]
```diff
@@ -6688,11 +6688,17 @@ export namespace SceneUtils {
     export function attach(child: Object3D, scene: Scene, parent: Object3D): void;
 }
 
+interface Vec2
+{
```
Comment (efokschaner) [inline types/three/three-core.d.ts]: Nitpicking: rest of file is egyptian braces.

Comment (armanio123) [reply types/three/three-core.d.ts]: @FishOrBear could you please address the review comments?

Comment (efokschaner) [reply types/three/three-core.d.ts]: I think it's fine to merge but a style fix follow-up would be appreciated.

Comment (armanio123) [reply types/three/three-core.d.ts]: Ok, Ill merge the PR since it has been open for some time. The style can be addressed on another PR.

**Pull request merged**